### ISSUE TITLE
What does `es7` mean?

### ISSUE THREAD

**Issue by lacolaco** [gdi2290/es7-reflect-metadata#2]
I have a question. I couldn't understand what the name, `es7-reflect-metadata`, means. 

Draft of ES7 (should be called as ES2016) has been snapshotted already. ReflectMetadata never be included by ES7. 

https://twitter.com/bterlson/status/692424625813377026

I think this package is just a fork of `reflect-metadata` package for supporting UMD and Webpack. Why did you name this `es7-reflect-metadata`? 

Thank you.

**Comment by PatrickJS** [comment]
it's a fork of `reflect-metadata` and I just added the `es7` just because I wanted a version that worked with webpack without having to include the package in `noParse`. It's better universal support but I have to keep the repo in sync with `reflect-metadata`
https://github.com/gdi2290/es7-reflect-metadata/blob/master/package.json#L6...L14

**Comment by lacolaco** [comment]
Yes, I know the purpose of this fork and I think it's reasonable and necessary. But sorry, to be honest, I want you to change the incorrect name before angular depended on this.

For example in angular, `es6-promise` and `es6-collections` are the feature of ECMAScript 6 actually.
But `es7-reflect-metadata` is not similar meaning. 

I think this package should be named `universal-reflect-metadata` or `reflect-metadata-universal`.

**Comment by PatrickJS** [comment]
yup, `universal-reflect-metadata` makes more sense

**Comment by lacolaco** [comment]
Thank you for understanding. :smiley:
